lib/node_modules/@stdlib/stats/base/dists/binomial/cdf/benchmark/benchmark.js

```diff
@@ -21,64 +21,83 @@
 // MODULES //
 
 var bench = require( '@stdlib/bench' );
-var ceil = require( '@stdlib/math/base/special/ceil' );
+var Float64Array = require( '@stdlib/array/float64' );
+var uniform = require( '@stdlib/random/array/uniform' );
 var randu = require( '@stdlib/random/base/randu' );
+var floor = require( '@stdlib/math/base/special/floor' );
 var isnan = require( '@stdlib/math/base/assert/is-nan' );
 var pkg = require( './../package.json' ).name;
 var cdf = require( './../lib' );
 
 
 // MAIN //
 
-bench( pkg, function benchmark( b ) {
+bench( pkg, function benchmark( b )
+{
+	var len;
+	var k;
 	var n;
 	var p;
-	var x;
 	var y;
 	var i;
 
+	len = 100;
+	k = new Float64Array( len );
+	n = new Float64Array( len );
+	p = new Float64Array( len );
+	for ( i = 0; i < len; i++ )
+	{
+		k[ i ] = floor( randu() * 11 );
+		n[ i ] = floor( randu() * 21 );
+		p[ i ] = randu();
+	}
+
 	b.tic();
-	for ( i = 0; i < b.iterations; i++ ) {
-		x = randu() * 100.0;
-		n = ceil( randu()*100.0 );
-		p = randu();
-		y = cdf( x, n, p );
-		if ( isnan( y ) ) {
+	for ( i = 0; i < b.iterations; i++ )
+	{
+		y = cdf( k[ i % len ], n[ i % len ], p[ i % len ] );
+		if ( isnan( y ) )
+		{
 			b.fail( 'should not return NaN' );
 		}
 	}
 	b.toc();
-	if ( isnan( y ) ) {
+	if ( isnan( y ) )
+	{
 		b.fail( 'should not return NaN' );
 	}
 	b.pass( 'benchmark finished' );
 	b.end();
-});
+} );
```

Comment:
```suggestion
});
```

Not clear to me why this PR contains unnecessary style changes.